Xamarin.Forms.Platform.Tizen/Platform.cs

```diff
@@ -120,6 +120,7 @@ internal DefaultPlatform(EvasObject parent)
 			MessagingCenter.Subscribe<Page, bool>(this, Page.BusySetSignalName, BusySetSignalNameHandler);
 			MessagingCenter.Subscribe<Page, AlertArguments>(this, Page.AlertSignalName, AlertSignalNameHandler);
 			MessagingCenter.Subscribe<Page, ActionSheetArguments>(this, Page.ActionSheetSignalName, ActionSheetSignalNameHandler);
+			MessagingCenter.Subscribe<Page, PromptArguments>(this, Page.PromptSignalName, OnPromptRequested);
```

Comment:
This change will conflict with #8845. I don't know which one will merge first, but I will rebase whatever it is.